blog-test/migration.js

```diff
@@ -0,0 +1,95 @@
+import fs from 'fs';
+import { u } from 'unist-builder';
+import { BulkUpdate, ExcelReporter, loadListData, saveDocument } from '../bulk-update/index.js';
+import { selectBlock } from '../bulk-update/migration-tools/select.js';
+
+const { pathname } = new URL('.', import.meta.url);
+const dateString = ExcelReporter.getDateString();
+
+const config = {
+  list: [
+    'https://main--bacom-blog--adobecom.hlx.live/de/blog/query-index.json',
+    'https://main--bacom-blog--adobecom.hlx.live/fr/blog/query-index.json',
+    'https://main--bacom-blog--adobecom.hlx.live/au/blog/query-index.json',
+    'https://main--bacom-blog--adobecom.hlx.live/uk/blog/query-index.json',
+    'https://main--bacom-blog--adobecom.hlx.live/blog/query-index.json',
+    'https://main--bacom-blog--adobecom.hlx.live/jp/blog/query-index.json',
+    'https://main--bacom-blog--adobecom.hlx.live/kr/blog/query-index.json',
+  ],
+  siteUrl: 'https://main--bacom-blog--adobecom.hlx.live',
+  stagePath: '/drafts/staged-content',
+  locales: JSON.parse(fs.readFileSync(`${pathname}locales.json`, 'utf8')),
+  prodSiteUrl: 'https://business.adobe.com',
+  reporter: new ExcelReporter(`${pathname}reports/blog-${dateString}.xlsx`, false),
+  outputDir: `${pathname}output`,
+  mdDir: `${pathname}md`,
+  mdCacheMs: 1 * 24 * 60 * 60 * 1000, // 1 day(s)
+  fetchWaitMs: 20,
+};
+
+/**
+ * Creates a block with the given name and fields.
+ *
+ * @param {string} name - The name of the block.
+ * @param {Object} fields - The fields of the block.
+ * @returns {Array} - The created block.
+ */
+export function createBlock(name, fields) {
+  const block = u('gridTable', [
+    u('gtBody', [
+      u('gtRow', [
+        u('gtCell', { colSpan: 2 }, [u('paragraph', [u('text', name)])]),
+      ]),
+      ...fields.map((values) => u('gtRow', values.map((value) => u('gtCell', [u('paragraph', [value ?? u('text', '')])])))),
+    ]),
+  ]);
+
+  return block;
+}
+
+/**
+ * Create or replace the hide block with the entry and date
+ *
+ * @param {Object} document - The document to be migrated.
+ */
+export async function migrate(document) {
+  const { mdast, entry } = document;
+  if (!mdast || !mdast.children) return;
+
+  const fields = [
+    [u('text', 'Entry'), u('text', entry)],
+    [u('text', 'Date'), u('text', new Date().toISOString().split('T')[0])],
+  ];
+  const hideBlock = createBlock('Hide Block', fields);
```

Comment:
It's a little confusing to name the block with a verb. As in, is it really hiding something or is it hidden? Might be more aptly name `Hidden Block` if it's the latter. If this is just a test, it might not be important though.

Reply:
This block is display none in milo. I wanted to add a block like we did with card metadata but did not want it change anything visually.
https://github.com/adobecom/milo/blob/bmarshal/marketo-localized/libs/styles/styles.css#L723-L725